README.md

```diff
@@ -5,6 +5,15 @@
 This extension adds language support for [Haskell](https://haskell.org), powered by the [Haskell Language Server](https://github.com/haskell/haskell-language-server).
 As almost all features are provided by the server you might find interesting read its [documentation](https://haskell-language-server.readthedocs.io).
 
+## Setup
+
+This Extension comes with "batteries"-included and can manage your Haskell Language Server installations for you,
+powered by [GHCup](https://www.haskell.org/ghcup/).
+Installation of [GHCup](https://www.haskell.org/ghcup/) can not happen automatically, so if you want your HLS installations to be
+managed by the Extension, you will have to follow the [installation instructions for GHCup](https://www.haskell.org/ghcup/).
```

Comment:
@hasufell This is now the first thing new users see, I don't think that's great, but better than nothing